tvshows/src/app/shows/[id]/page.tsx

```diff
@@ -0,0 +1,10 @@
+'use client';
+import { ShowReviewSection } from '../../../components/features/shows/ShowReviewSection/ShowReviewSection';
+
+const ShowDetails = () => {
+  return (
+    <ShowReviewSection/>
```

Comment:
I mentioned this in another comment, but I would expect this page to contain a ShowContainer or something similar. Not really a ShowReviewSection, that makes me think all I see are reviews.